boards/arm/nucleo_f429zi/nucleo_f429zi.dts

```diff
@@ -96,3 +97,51 @@ arduino_serial: &usart6 {};
 &iwdg {
 	status = "ok";
 };
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * http://docs.zephyrproject.org/devices/dts/flash_partitions.html
+	 */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* 128K for bootloader */
```

Comment:
Isn't it a bit large

Reply:
Yes, but the first several sectors are like below:
```
 * sector 0     16K   0x000 0000
 * sector 1     16K
 * sector 2     16K
 * sector 3     16K
 * sector 4     64K
```

So, either using not more than 64K or 128K. 64K is a bit tight.

Reply:
ok